maps/map_files/USS_Almayer/USS_Almayer.dmm

```diff
@@ -9046,7 +9046,11 @@
 "blw" = (
 /obj/structure/surface/table/almayer,
 /obj/item/tool/lighter,
-/obj/structure/machinery/faxmachine/uscm,
+/obj/structure/machinery/faxmachine/uscm{
+	desc = The beloved fax machine of the Requisitions department. Be sure to make a convincing argument when you ask high command for a crate of lunge mines.;
```

Comment:
quotation marks required for text strings. Repeat for other errors. Also no need to reference lunge mines.